third_party/lua/luadebug.lua

```diff
@@ -27,6 +27,7 @@
 local dbg
 
 local DEBUGGER = 'luadebug'
+local HISRTORYFILE = '.tdbg_history'
```

Comment:
Nit: Should it actually be named as '**hisrtory** file'? :)

Reply:
Yep, fixed.